rero_ils/utils.py

```diff
@@ -42,15 +43,18 @@ def send_mail(subject, recipients, template, language, **context):
                   sender=sender,
                   recipients=recipients)
 
-    ctx = ('email', template, language)
-
     if config_value('EMAIL_PLAINTEXT'):
-        msg.body = render_template('%s/%s_%s.txt' % ctx, **context)
+        msg.body = render_template(
+            'email/{template}_{language}.txt'.format(
+                template=template,
+                language=language
+            ),
+            **context
+        )
     # if config_value('EMAIL_HTML'):
     #     msg.html = render_template('%s/%s.html' % ctx, **context)
 
-    mail = current_app.extensions.get('mail')
-    mail.send(msg)
+    send_mail_async.delay(msg.__dict__)
```

Comment:
can you use `invenio-mail` as in the Dispatcher?

Reply:
This is the `send_mail` from `invenio-mail`.